packages/govern/internal/actions/DAOSAction.ts

```diff
@@ -0,0 +1,17 @@
+import { DocumentNode } from 'graphql'
+import AbstractAction from './lib/AbstractAction'
+import daos from '../clients/graphql/queries/daos'
+
+/**
+ * @class DAOSAction
+ */
+export default class DAOSAction extends AbstractAction {
```

Comment:
```suggestion
export default class DaosAction extends AbstractAction {
```

Reply:
I will do that.. that's why asked for the Aragon Code Style document.